jenkins/PerfCI_Chaos/03_PerfCI_Choas_Windows_VMs_Deployment/Jenkinsfile

```diff
@@ -197,8 +227,8 @@ END
                                         -e IBM_BUCKET='${IBM_BUCKET}' \
                                         -e IBM_KEY='${IBM_KEY}' \
                                         -e GOOGLE_DRIVE_PATH='${GOOGLE_DRIVE_PATH}' \
-                                        -e GOOGLE_DRIVE_CREDENTIALS_FILE='${GOOGLE_DRIVE_CREDENTIALS_FILE}' \
-                                        -e GOOGLE_DRIVE_TOKEN_FILE='${GOOGLE_DRIVE_TOKEN_FILE}' \
+                                        -e GOOGLE_DRIVE_CREDENTIALS_FILE='${DESTINATION_PATH}/${CREDENTIALS_FILE}' \
+                                        -e GOOGLE_DRIVE_TOKEN_FILE='${DESTINATION_PATH}/${TOKEN_FILE}' \
```

Comment:
Are DESTINATION_PATH and CREDENTIALS_FILE constant, or do they change from run to run (so that if e. g. you're doing two concurrent runs they would not collide)?

Reply:
Every run those files are deleted at the end.

Reply:
1. If they're fixed file names, someone could guess them.
2. What happens if there are multiple concurrent runs?

They really should be distinct random filenames, or at least within distinct randomly generated directory names.

Reply:
Non-colliding directory names that are mode 700 would be fine.

Reply:
Sorry Robert, I am not going to spend more time on Jenkins.

Reply:
OK, then I'll open an issue to record it and approve this.